src/base/matrix.rs

```diff
@@ -53,8 +53,7 @@ pub type VectorSum<T, R1, R2> =
     Matrix<T, SameShapeR<R1, R2>, U1, SameShapeStorage<T, R1, U1, R2, U1>>;
 
 /// The type of the result of a matrix cross product.
-pub type MatrixCross<T, R1, C1, R2, C2> =
-    Matrix<T, SameShapeR<R1, R2>, SameShapeC<C1, C2>, SameShapeStorage<T, R1, C1, R2, C2>>;
+pub type MatrixCross<T, R, C> = Matrix<T, R, C, Owned<T, R, C>>;
```

Comment:
By the way, I'd like to ask some questions. Why do we need this type alias? I don't think it simplifies much.